### PR TITLE
set up i18n

### PR DESCRIPTION
**Pull request by dkhgh**

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1,15 +1,15 @@
 <div>
+  <h1><%= t("teams.posts_for") %> for <%= current_team.name %></h1>
```
Comment (fatboypunk) [inline app/views/posts/index.html.erb:2]: Think we want to translate the `for` as well

File: app/views/teams/edit.html.erb [hunk 2]
```diff
@@ -1,3 +1,3 @@
-<h1>EDIT</h1>
+<h1><%= edit %></h1>
```
Comment (fatboypunk) [inline app/views/teams/edit.html.erb:1]: Does this work?

File: app/views/teams/show.html.erb [hunk 3]
```diff
@@ -19,8 +20,8 @@
 
   <p>
     <% if is_member %>
-      <%= link_to "Leave Team :(", leave_team_path(@team), method: :post, class: "button" %>
+      <%= button_to t("teams.leave_team"), leave_team_path(@team), class: "button" %>
```
Comment (fatboypunk) [inline app/views/teams/show.html.erb:23]: We need some css to remove the class button for buttons

File: app/views/posts/index.html.erb (hunk 1 above)
Comment (dkhgh) [inline app/views/posts/index.html.erb:2]: true

File: app/views/teams/edit.html.erb (hunk 2 above)
Comment (dkhgh) [inline app/views/teams/edit.html.erb:1]: nope:

screen shot 2015-11-03 at 12 11 01

Comment (dkhgh) [inline app/views/teams/edit.html.erb:1]: luckily there's no button to team_edit_path yet :dash:

**Pull request merged**